lib/node_modules/@stdlib/blas/ext/base/ssorthp/benchmark/benchmark.mostly_sorted_few_uniques.js

```diff
@@ -60,7 +59,7 @@ function createBenchmark( iter, len ) {
 		for ( j = 0; j < len; j++ ) {
 			sa = (b-a) * (j/len);
 			sb = sa / 2.0;
-			tmp[ j ] = floor( uniform( a+sa, b+sb ) );
+			tmp[ j ] = 2*( uniform( a+sa, b+sb ) )>>1;
```

Comment:
This isn't where you should be removing "floor". You are only supposed to remove from documentation and tests. E.g.,

```
N = floor( x.length / 2 );
```

Those are the only instances. Not here in the benchmarks.